src/components/EntitlementsAgreementsList/EntitlementAgreementsList.js

```diff
@@ -29,6 +30,18 @@ const EntitlementAgreementsList = (
     totalCount,
     visibleColumns }
 ) => {
+  const { settings } = useAgreementsSettings();
+  const entitlementAgreementsPageSize = parseMclPageSize(settings, 'entitlementOptions');
```

Comment:
If I'm reading it properly,, this should be the setting `entitlements` not `entitlementOptions`. Our naming is not ideal though... easy mistake to make

Reply:
Ah, and I believe the other one is also flipped